source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,116 @@
+
+1. Get started as follows:
+
+    ```
+    $ minishift start
+     Starting local OpenShift cluster using 'kvm' hypervisor...
+     ...
+        OpenShift server started.
+        The server is accessible via web console at:
+            https://192.168.99.128:8443
+
+        You are logged in as:
+            User:     developer
+            Password: developer
+
+        To login as administrator:
+            oc login -u system:admin
+    ```
+
+    This will:
+    - Download the latest [_minishift-b2d-iso_](https://github.com/Minishift/minishift-b2d-iso) (~40 MB)
+    - Starts a VM using libmachine
+    - Download OpenShift client binary (`oc`) with version `1.4.1`
+    - Cache both `oc` and ISO into your `$HOME/.minishift/cache` folder
+    - Finally provisions OpenShift single node cluster
+
+    **Note:** This step also logs you in as `developer` in the `myproject` project.
+
+1. Export the OpenShift client binary (`oc`) to the _PATH_ environment variable as follows:
+
+    ```
+    $ export PATH=$PATH:$HOME/.minishift/cache/oc/v1.4.1
+    ```
+
+1. Deploy a sample application as follows:
+    - Create a Nodejs example application:
+
+         ```
+         $ oc new-app https://github.com/openshift/nodejs-ex -l name=myapp
+         ```
+    - Track the build log until the app is built and deployed using:
+
+        ```
+        $ oc logs -f bc/nodejs-ex
+        ```
+    - Expose a route to the service as follows:
+
+        ```
+        $ oc expose svc/nodejs-ex
+        ```
+    - Open the application in a browser as follows:
+
+        ```
+        $ minishift openshift service nodejs-ex -n myproject
+        ```
+
+1. When done with building the application for the day, stop the cluster as follows:
+
+    ```
+     $ minishift stop
+     Stopping local OpenShift cluster...
+     Stopping "minishift"...
+    ```
+
+    If you want to access the same cluster some other day, use `minishift start`.
+
+1. At anytime if you want to delete the cluster, use the following command:
+
+    ```
+    $ minishift delete
+    ```
+
+Refer to the [Minishift README](https://github.com/Minishift/minishift#minishift) to get more detailed description of getting started with a single node OpenShift cluster.
+
+## Feedback
+
+Try Minishift!
+
+We would love to get your feedback. If you hit a problem, please raise an issue in [minishift issue tracker](https://github.com/minishift/minishift/issues) but please search through the listed issues before creating it. It's possible that a similar issue is already open.
+
+You could also reach us at our IRC channel as mentioned below.
+
+## Community
+
+The Minishift community hangs out on the IRC channel `#minishift` on Freenode (https://freenode.net). You are welcome to join, participate in the discussions, and contribute.
+
+## Resources
+- Github Repo: https://github.com/minishift
+- Documentation: https://github.com/minishift/minishift#documentation
+- OpenShift Origin: https://www.openshift.org/minishift
```

Comment:
This link description is wrong. It is not the link for OpenShift Origin. Sorry for not noticing in previous reviews.